lib/ruby_parser_string_io_patch.diff

```diff
@@ -2,17 +2,17 @@
 +++ lib/ruby_lexer.rb	2008-04-27 01:07:03.000000000 +0200
```

Comment:
Now I'm reading through this and wonder why this file is even here. I think it can be dropped. It was added in eb79570c2794415cc3d5d960cccb7ccda08f136f.